CHANGELOG.md

```diff
@@ -0,0 +1,236 @@
+
+Release date: 2019-07-03
+
+-   Test fixes.
+
+### 2.19 
+
+Release date: 2018-05-17
+
+-   Bugfix: builds failing upon running a JobPropertyStep with error
+    "did not yet start" if previous build was hard-killed or failed
+    before Pipeline began running
+    ([JENKINS-51290](https://issues.jenkins-ci.org/browse/JENKINS-51290))
+
+### 2.18 
+
+Release date: 2018-04-20
+
+-   Correct small bug from the improper use of the
+    FlowExecutionOwner.getOrNull() call for getting the previous
+    Properties for a step (visible as a result of
+    [JENKINS-50784](https://issues.jenkins-ci.org/browse/JENKINS-50784))
+
+### 2.17 
+
+Release date: 2018-01-22
+
+-   Provide a BranchProperty to support setting configuring durability
+    per-branch (supports
+    [JENKINS-47300](https://issues.jenkins-ci.org/browse/JENKINS-47300))
+-   Massive updates to dependencies and tests
+-   Bugfix: [JENKINS-42817](https://issues.jenkins-ci.org/browse/JENKINS-42817)
+    readTrusted Pipeline from SCM only works in \*/master
+
+### 2.16
+
+Release date: 2017-06-19
+
+-   Bugfix: Stop removing JobProperties defined outside a step
+    ([JENKINS-44848](https://issues.jenkins-ci.org/browse/JENKINS-44848))
+
+### 2.15
+
+Release date: 2017-06-01
+
+-   [JENKINS-34561](https://issues.jenkins-ci.org/browse/JENKINS-34561) Option
+    to select a script name/path other than `Jenkinsfile`.
+
+### 2.14
+
+Release date: 2017-03-10
+
+-   [JENKINS-40558](https://issues.jenkins-ci.org/browse/JENKINS-40558) Replace
+    in-line help references of "pipeline script" with "pipeline" to
+    clarify that [Pipeline Model Definition
+    Plugin](https://plugins.jenkins.io/pipeline-model-definition) works
+    with multibranch
+
+### 2.13
+
+Release date: 2017-03-03
+
+-   [JENKINS-33273](https://issues.jenkins-ci.org/browse/JENKINS-33273)
+    Load `Jenkinsfile` (or any file requested in `loadTrusted`) directly
+    from the SCM rather than doing a checkout. Requires a compatible SCM
+    (currently Git or GitHub).
+-   [JENKINS-40521](https://issues.jenkins-ci.org/browse/JENKINS-40521)
+    Orphaned branch projects are now effectively disabled.
+-   [JENKINS-41146](https://issues.jenkins-ci.org/browse/JENKINS-41146)
+    Help text improvement.
+-   API change: `AbstractWorkflowMultiBranchProjectFactory` implementers
+    may now throw `IOException`.
+
+One aspect of
+[JENKINS-33273](https://issues.jenkins-ci.org/browse/JENKINS-33273) is
+that you will not get an SCM changelog from a lightweight checkout of
+`Jenkinsfile` itself---only if your script actually runs `checkout scm`.
+
+This behavior change can be suppressed in case of emergency using
+`-Dorg.jenkinsci.plugins.workflow.multibranch.SCMBinder.USE_HEAVYWEIGHT_CHECKOUT=true`.
+
+### 2.12
+
+Release date: 2017-02-02
+
+-   [JENKINS-40906](https://issues.jenkins-ci.org/browse/JENKINS-40906) Add
+    missing help text and configuration UI
+
+### 2.10
+
+Release date: 2017-01-16
+
+-   ![(warning)](docs/images/warning.svg)
```

Comment:
@sladyn98 This is where the warning image was used. I would delete this, then pull the next line up and make it bold.
```suggestion
-   
```